### DISCUSSION TITLE
点子0  精神图腾NFT  将每篇文章做成NFT

### DISCUSSION THREAD

**Discussion by happybig** [famland/website#68] [Ideas]
我又有一个新创意:

既然famland的精神图腾是topal老师。为什么不设计一个更好玩的玩法:  文章NFT

打造一种特殊的"精神图腾NFT"，每一个唯一的NFT对应topal的一篇经典文章。这些文章放在官网上并标上唯一的标识码，将文章标题及标识码印在NFT上，NFT图标显著显示标题。NFT附带一些功能参数值。这些参数值可赋予相应功能。其中一个参数值可以关连，该文章在官网上所获得的点赞数量。比如参数值A的功能可影响famland收益率(或者其他什么东东)，而文章每获点赞数增加1，则参数A加1。或者还可以有更高级的赋能方法: 由topal签署授权证明:此文的版权授予famland系统平台，平台同时又转授权给该NFT的所有者。未来如果这些文章有版权收入就由平台负责收取，并全部转付给该NFT所有者。

如果此思路走通了，将是一个大大的突破。石破天惊的突破!

其实实现办法也很简单，
官网开通一个，授权发布文章的功能，即所有人可以将自己的文章发布在官网上，发布文章同时就自动签署了版权授权协议书，协议规定将版权授予给famland官网平台，由官网平台再将版权授予NFT所有者。

应该是没有总哈希值。而且你提到的把点赞数作为提升NFT属性，好像实现难度会很大

其实不用这么复杂。先只要实现文章关连进NFT，就是一个大大的卖点

比如我进我的EOS钱包可以看到一个NFT"我有一个梦想"，(在famland系统内)点击这个NFT可以跳转链接打开这个文章，还可查看关于这篇文章的一些属于数据。就太酷了

满足每个人自己的虚荣心。

如果这个NFT还可以自由转帐，自由交易，或者在平台反复拍卖。多吸引人呀

**Comment by happybig** [comment]
很好

**Reply by yanzhiwuli** [reply]
你的想法很棒，在FAMLAND完全可以实现你的 想法。
